src/brouwers/kitreviews/models.py

```diff
@@ -108,5 +104,23 @@ class Meta:
 
     def __unicode__(self):
         return _(u"Vote for review by %(review_submitter)s") % {
-                    'review_submitter': get_username(self.kit_review, field='reviewer')
-                    }
+            'review_submitter': get_username(self.kit_review, field='reviewer')
+        }
+
+
+class KitReviewProperty(models.Model):
+    """ Model containing the properties of a review, which based on their score are divided into pros and cons"""
+
+    name = models.CharField(_(u'name'), max_length=255)
+    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
```

Comment:
This should move to the `through` model of the relation, see above. Sorry for not noticing earlier.